Products/ZenUtils/ShardedBTree.py

```diff
@@ -12,6 +12,8 @@
 from itertools import chain
 from zlib import adler32
 
+import logging
+log = logging.getLogger('zen.GuidManager')
```

Comment:
Please name the logger something else. Currently we only use the sharded btree for the guid table but in the future we can use it to store any other data.